src/components/Highlight.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-  <span
-    :class="suit()"
-    v-html="innerHTML"
+  <highlighter
```

Comment:
A lowercase component without split isn’t valid. We can call this AisHighlighter

Reply:
https://github.com/algolia/vue-instantsearch/pull/783/commits/395b9856f3c3f9927957e02d98a63f3d7cf04fdd